modules/nf-core/affy/justrma/environment.yml

```diff
@@ -5,3 +5,4 @@ channels:
   - defaults
 dependencies:
   - bioconda::bioconductor-affy=1.78.0
+  - conda-forge::r-base=4.3.1
```

Comment:
Doesn't that come with a bioconductor package!? Or is it unpinned in conda?

Reply:
Correct, it is unpinned in conda, was using 4.3.3 in the conda recipe.